ionoscloud/resource_vpn_ipsec_gateway.go

```diff
@@ -50,8 +50,9 @@ func resourceVpnIPSecGateway() *schema.Resource {
 				Type:        schema.TypeList,
 				Description: "The network connection for your gateway. Note: all connections must belong to the same datacenter.",
 				MinItems:    1,
-				MaxItems:    10,
-				Required:    true,
+				// TODO -- Change this from 10 to 5 or leave this validation for the API
```

Comment:
do we need to change to 5?

Reply:
@cristiGuranIonos the `maxItems` it's now 5 in the swagger. I added this `TODO` but I'm not sure about this change, I think it's better to simply remove this validation and let the API do all the validation.